index.bs

```diff
@@ -7096,9 +7105,38 @@ Note that when |nonce| is empty, then the (signed) authenticator extension outpu
 
 Verifying the <code>[=devicePubKey=]</code> extension output is performed by the [=[RP]=] whenever a [=device public key=] is returned within the extension output.
 
+The [=devicePubKey=] extension adds the following [=struct/item=] to [=credential records=]:
+
+<dl dfn-for="credential record" dfn-type="abstract-op">
+    :   <dfn>devicePubKeys</dfn>
+    ::  A (possibly [=set/empty=]) [=set=] of [=device-bound key records=] associated with this [=public key credential source=].
+
+        A <dfn dfn>device-bound key record</dfn> is an abstract representation of a registered [=device-bound key=].
+        It is a [=struct=] with the following [=struct/items=]:
+
+        <dl dfn-for="devicePubKey record" dfn-type="abstract-op">
+            :   <dfn>aaguid</dfn>
+            ::  The [=AAGUID=] of the [=device-bound key=]'s [=managing authenticator=].
+                This MAY be different from the [=AAGUID=] in the [$credential record/attestationObject$], if any, of the containing [=credential record=].
+
+            :   <dfn>dpk</dfn>
+            ::  The public key portion of the [=hardware-bound device key pair=].
```

Comment:
Here `hardware-bound device key pair` is used, but `device-bound key` is used in other places (7112, 7126, 7129, 7132). Those could be interpreted as having different meanings. I know the terms section declares both for the definition, but it may still be confusing.

Reply:
Hmyeah, I can see that. Maybe we just use `[=device-bound key=]` here too then?

Reply:
That works!